docs/specs/cafcass-search-cases.json

```diff
@@ -473,6 +473,188 @@
+                    "domesticAbuseDocument": {
+                      "$ref": "#/components/schemas/domesticAbuseDocument"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "mpuUrgencyReason": {
+            "type": "string",
+            "example": "miamPolicyUpgradeUrgencyReason_Value_1"
+          },
+          "mpuPreviousMiamAttendanceReason": {
+            "type": "string",
+            "example": "miamPolicyUpgradePreviousAttendance_Value_2"
+          },
+          "mpuTypeOfPreviousMiamAttendanceEvidence": {
+            "type": "string",
+            "example": "miamCertificate"
+          },
+          "mpuCertificateByMediator": {
+            "type": "object",
+            "properties": {
+              "document_id": {
+                "type": "string",
+                "format": "uuid",
+                "example": "zzz48b24-c5c2-451c-9fd5-d7a6a87c6355"
+              },
+              "document_filename": {
+                "type": "string",
+                "example": "miamCertificate.pdf"
+              }
+            }
+          },
+          "mpuOtherExemptionReasons": {
+            "type": "string",
+            "example": "miamPolicyUpgradeOtherGrounds_Value_3"
+          },
+          "mpuApplicantUnableToAttendMiamReason1": {
+            "type": "string",
+            "example": "mpuChildProtectionConcern_value_1"
+          },
+          "mpuApplicantUnableToAttendMiamReason2": {
+            "type": "string",
+            "example": "mpuChildProtectionConcern_value_2"
+          },
+          "mpuDocFromDisputeResolutionProvider": {
+            "type": "object",
+            "properties": {
+              "document_id": {
+                "type": "string",
+                "format": "uuid",
+                "example": "zzz48b24-c5c2-451c-9fd5-d7a6a87c6355"
+              },
+              "document_filename": {
+                "type": "string",
+                "example": "disputeResolution.pdf"
+              }
+            }
+          },
+          "miamCertificationDocumentUpload": {
+            "type": "object",
+            "properties": {
+              "document_id": {
+                "type": "string",
+                "format": "uuid",
+                "example": "zzz48b24-c5c2-451c-9fd5-d7a6a87c6355"
+              },
+              "document_filename": {
+                "type": "string",
+                "example": "certificateDocument.pdf"
+              }
+            }
+          },
+          "mpuChildProtectionConcernReason": {
+            "type": "string",
+            "example": "child protection concern"
+          },
+          "mpuMediatorDetails": {
+            "type": "string",
+            "example": "mediator details"
+          },
+          "mpuNoDomesticAbuseEvidenceReason": {
+            "type": "string",
+            "example":"domestic abuse evidence reason"
+          },
+          "miamPolicyUpgradeTable": {
```

Comment:
what is the relevance of this field to the api?

Reply:
@yogendraU in relevance with miam table which was already there that is why kept there.

Reply:
Please remove this field